src/users/dto/user.dto.ts

```diff
@@ -1,4 +1,6 @@
-import { IsEmail, IsString, MinLength } from 'class-validator';
+import { Type } from 'class-transformer';
+import { IsArray, IsEmail, IsNumber, IsOptional, IsString, MinLength } from 'class-validator';
+import { AssignCourseDto } from 'src/courses/dto/courses.dto';
```

Comment:
itt ez az src alóli automatikus abszolút url-es import a VSCode által generált, bár működik, én a saját importot én itt inkább konzisztensen relatív útvonallal húznám be, hátha valahol gondotot okoz szerveren hogy nem tudja feloldani ezeket build után